setup.py

```diff
@@ -4,11 +4,14 @@
 from setuptools import setup, find_packages
 from os         import path
 
+import cvmfs
+
+
 readme_path = path.join(path.dirname(__file__), 'README')
 
 setup(
-  name="python-cvmfsutils",
-  version="0.2.0",
+  name=cvmfs.package_name,
+  version=cvmfs.version,
```

Comment:
well, I'd like to allow users to query the library version they are using at runtime...
Another idea:
Could we put the `__version__` and `__package_name__` into a separate file (i.e. `__meta_info__.py`) and include that into the `setup.py` directly? If we are lucky, Python doesn't silently include `__init__.py` and we could live happily ever after.